### PR TITLE
[Improvement-11507][Workflow] support sort for workfow definition list  

### LINKED ISSUE #11507: [Improvement][Workflow] support sort for workfow definition list
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

change api: /dolphinscheduler/projects/{projectCode}/process-definition
add enum:
the rules are as follow：
when sortEnum == CreateTimeDesc generate order by create_time desc
when sortEnum == CreateTimeAsc generate order by create_time asc
when sortEnum == UpdateTimeDesc generate order by update_time desc
when sortEnum == UpdateTimeAsc generate  order by update_time asc

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by 106umao**
close #11507
## Brief change log
close #11507

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/SortEnum.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.apache.dolphinscheduler.api.enums;
+
+public enum SortEnum {
```
Comment (lenboo) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/SortEnum.java:21]: no comments

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java [hunk 2]
```diff
@@ -491,6 +492,7 @@ public Result queryProcessDefinitionListPaging(@ApiIgnore @RequestAttribute(valu
                                                    @RequestParam(value = "searchVal", required = false) String searchVal,
                                                    @RequestParam(value = "otherParamsJson", required = false) String otherParamsJson,
                                                    @RequestParam(value = "userId", required = false, defaultValue = "0") Integer userId,
+                                                   @RequestParam(value = "sortEnum", required = false) SortEnum sortType,
```
Comment (lenboo) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java:495]: add default value

File: dolphinscheduler-standalone-server/src/main/resources/logback-spring.xml [hunk 3]
```diff
@@ -78,6 +78,7 @@
                 <appender-ref ref="STDOUT"/>
             </then>
         </if>
+        <appender-ref ref="STDOUT"/>
```
Comment (lenboo) [inline dolphinscheduler-standalone-server/src/main/resources/logback-spring.xml:81]: remove this unused code

**Review by lenboo — CHANGES_REQUESTED** [review]
some need changed

**Comment by 106umao** [comment]
close

**Pull request closed without merging**